code/modules/admin/verbs/serialization.dm

```diff
@@ -24,3 +24,5 @@
 	var/json_text = input("Enter the JSON code:","Text") as message|null
 	if(json_text)
 		json_to_object(json_text, get_turf(usr))
+		message_admins("[key_name_admin(usr)] spawned a custom JSON datum.")
+		log_admin("[key_name(usr)] created a custom JSON datum: [json_text]")
```

Comment:
```suggestion
		message_admins("[key_name_admin(usr)] spawned an atom from a custom JSON object.")
		log_admin("[key_name(usr)] spawned an atom from a custom JSON object, JSON Text: \"[json_text]\"")
```
few phrasing nitpicks, this will make it much more clear to admins what this log is actually indicating. The admin is spawning in items/mobs by deserializing a JSON object and isn't actually creating datums.